app/views/help/index.html.haml

```diff
@@ -0,0 +1,12 @@
+%ul
+  - @pages.each do |page|
+    - # this is a hack to capture the :title from each page: render the template, without actually outputting it
+    - render template: "help/#{page}"
```

Comment:
What do you think about doing this pre-rendering step inside the controller instead of the view? Then `@pages` could be a list of `(path, title)` pairs? Ideally the controller would be able to fetch the meta data without rendering, but I don't see a simple way to do that either.